Explorer/Assets/DCL/PerformanceAndDiagnostics/PerformanceBudgeting/BudgetingConfig.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace DCL.PerformanceBudgeting
+{
+    public static class BudgetingConfig
+    {
+        // FPS
+        public const int FPS_CAP = 40; // [ms]
+
+        // Concurrent assets loading
+        public const int SCENES_LOADING_BUDGET = 100;
+        public const int ASSETS_LOADING_BUDGET = 50;
+
+        // Memory
+        public static readonly Dictionary<MemoryUsageStatus, float> MEM_THRESHOLD = new ()
```

Comment:
Let's move it to the plugin's settings: other constants were there before we developed the scheme with plugins & settings

Reply:
I think it good to keep all this budgeting settings in one place (for easier tweaking).  And it is not so much related to plugins, but rather to Providers. For example, `FPS_CAP`, `ASSETS_LOADING_BUDGET `(and potentially `MEM_THRESHOLD`, if we pass it through the constructor) called from `StaticContainer `for respectively `FrameTimeCapBudgetProvider `, `ConcurrentLoadingBudgetProvider` and `MemoryBudgetProvider`, while `SCENES_LOADING_BUDGET` is called from `GlobalWorldFactory` for provider for `GlobalDeferredLoadingSystem`

I don't see downsides of having it in the static class in this assembly, since it is fully readonly and const